python/ray/llm/_internal/serve/deployments/prefill_decode_disagg/prefill_decode_disagg.py

```diff
@@ -151,7 +151,7 @@ async def _handle_request(
         prefill_request = self._prepare_prefill_request(request)
         prefill_gen = getattr(self.prefill_server, method).remote(prefill_request)
 
-        prefill_chunk = await anext(prefill_gen)
+        prefill_chunk = await prefill_gen.__anext__()
```

Comment:
```suggestion
        prefill_chunk = await anext(prefill_gen)
```

Is the dunder method required here?

Reply:
this syntax apparently has some lint issues not sure why.